apps/idos-enclave/src/lib/enclave.js

```diff
@@ -31,6 +36,15 @@ export class Enclave {
 
     const storeWithCodec = this.store.pipeCodec(Base64Codec);
 
+    if (!this.isAuthorizedOrigin) {
+      return {
+        humanId: "",
+        encryptionPublicKey: "",
+        signerAddress: "",
+        signerPublicKey: "",
+      };
+    }
```

Comment:
Please no. This effectively creates a singleton value in the type's domain that all callers will have to test for. By turning this into a throw instead, we're making sure this is handled differently than a normal response more intently, and we don't pollute the type domain.

Can we make this raise instead?